pySDC/implementations/problem_classes/Burgers.py

```diff
@@ -5,6 +5,11 @@
 
 
 class Burgers1D(GenericSpectralLinear):
+    """
+    See https://en.wikipedia.org/wiki/Burgers'_equation
+    Discretization is done with a Chebychov method, which requires a first order derivative formulation.
+    Feel free to do a more efficient implementation using an ultraspherical method to avoid the first order business.
+    """
```

Comment:
Please don't cheat your way in : class documentation implies describing the parameters, so anyone using your class know what can be put in it ... Not sure the wikipedia link helps a lot here ...

Reply:
PS : see https://github.com/Parallel-in-Time/pySDC/blob/master/pySDC/implementations/problem_classes/odeScalar.py for an example.

Also, writing documentation is quite easy with Spyder (with auto generation and rendering ...). But maybe there is also a plugin to do that in Vim ...